autogen/agentchat/contrib/capabilities/context_handling.py

```diff
@@ -1,9 +1,18 @@
 import sys
-from termcolor import colored
-from typing import Dict, Optional, List
-from autogen import ConversableAgent
-from autogen import token_count_utils
+from typing import Dict, List, Optional
+from warnings import warn
+
 import tiktoken
+from termcolor import colored
+
+from autogen import ConversableAgent, token_count_utils
+
+warn(
+    "Context handling with TransformChatHistory is deprecated. "
```

Comment:
Shall we add a note to the corresponding notebook?

Reply:
Just to clarify, a note to deprecate the notebook or a note to tell the user about the new transform messages notebook?

Reply:
I meant that there is a notebook using the old capability. We can add a note in that notebook about this and a link to the new notebook too.